src/indexdata.py

```diff
@@ -52,6 +52,21 @@ def parse_arguments():
     parser.add_argument('-f','--feature_type',help='Extract featureType during ingestion (to be done automatically).', action='store_true')
     parser.add_argument('-r','--remove',help='Remove the dataset with the specified identifier (to be replaced by searchindex).')
     parser.add_argument('-2','--level2',help='Operate on child core.')
+
```

Comment:
Concerning these options, shouldn't some of these be taken directly from MMD and others from the configuration file? At least we should discuss how to approach this.